src/components/TeamMember/TeamMember.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import SocialLink,{socialClasses} from '../SocialLink';
+import './TeamMember.css';
+import placeholder from './img/placeholder.png';
+
+function TeamMember({ firstName, lastName, photoUrl, info, links }) {
+    const parseLinks = (linkElement) => {
+        let linkType = '';
+        console.log(linkElement);
+        if(typeof(linkElement) !== "object") {
+            console.log('its object');
+            for(let type of Object.keys(socialClasses))
+                if(linkElement.includes(type))
+                    linkType = type;
+            return {
+                type: linkType,
+                url: linkElement
+            }
+        }
+        else
```

Comment:
takie podejście do instrukcji warunkowych jest trochę "bugogenne" - bardzo możliwe, jak doda się kolejną linijkę, to otrzymamy nie do końca oczekiwany rezultat. Najlepiej się przed tym zabezpieczyć dopisując klamerki, czyli:
```jsx
} else {
  return linkElement;
}
```